types/meteor-sjobs/index.d.ts

```diff
@@ -0,0 +1,154 @@
+// Type definitions for Steve Jobs task scheduler for Meteor.js
+// Project: https://github.com/msavin/SteveJobs
+// Definitions by: Witold H <https://github.com/LinearMilk/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'meteor/msavin:sjobs' {
+    export interface Options {
+        /** Specify if the package should start automatically on Meteor.startup. */
+        autoStart?: boolean;
+        /** Specify if the package should retry failed jobs whenever a new server takes control. */
+        autoRetry?: boolean;
+        /** Specify if the package should automatically delete internal data (not job related). */
+        autoPurge?: boolean;
+        /**  Specify how often the package should check for due jobs. */
+        interval?: number;
+        /** Specify how long after server startup the package should start running. */
+        startupDelay?: number;
+        /** Specify how long the server could be inactive before another server takes on the master role.	*/
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/meteor-sjobs/index.d.ts;
  Line: 18;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.